### PR TITLE
NOISSUE - Add Dockerfile.experimental

### PR DESCRIPTION
**Pull request by manuio**
Signed-off-by: Manuel Imperiale <manuel.imperiale@gmail.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,9 @@
 ui:
 	docker build --tag=mainflux/ui -f docker/Dockerfile .
 
+ui-experimental:
+	docker build --tag=mainflux/ui-experimental -f docker/Dockerfile.experimental .
```
Comment (mteodor) [inline Makefile:5]: is the dot in the required?

File: docker/Dockerfile.experimental [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Stage 0, based on Node.js, to build and compile Angular
+FROM node:14.14.0 as node
+WORKDIR /app
+COPY package.json /app/
+RUN npm install
+COPY ./ /app/
+RUN npm run experimental:prod
```
Comment (mteodor) [inline docker/Dockerfile.experimental:7]: is this the only difference with default file?

Comment (manuio) [reply docker/Dockerfile.experimental:7]: For now yes

File: Makefile (hunk 1 above)
Comment (manuio) [reply Makefile:5]: It's the context

**Review by mteodor — APPROVED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**